### ISSUE TITLE
Do we need makeLog() in Reporter.java?

### ISSUE THREAD

**Issue by gtrianta1965** [gtrianta1965/ServiceWatcher#39]
I have that question because after refactoring with velocity it seems useless to me.

**Comment by gtrianta1965** [comment]
I am asking this because you put in the velocity context both serviceParameters and log which actually they are the same "arrays" 
I believe that velocity can loop through ServiceParameters array and there is no need to create a separate hashmap (log)

**Comment by KALALEX** [comment]
Yes, that is correct you can loop through it keeping a more simplistic style (code side), rather than what we have now. I did it that way because the "ultimate" reason that some one will use this feature is to get the log, so i thought that this was the fastest, easiest way to access the log, for someone who just wants the log. And also it's there because it was written pre velocity age.
